src/Control/Session.php

```diff
@@ -324,6 +324,12 @@ public function start(HTTPRequest $request)
                 }
 
                 session_start();
+
+                // Session start emits a cookie, but only if there's no existing session. If there is a session tied to
+                // this request, make sure the session is held for the entire timeout by refreshing the cookie age.
+                if ($this->requestContainsSessionId($request)) {
```

Comment:
We don't really have to send it here if the `$timeout` is undefined or `0`

Reply:
Good catch, have amended.